README.md

```diff
@@ -0,0 +1,71 @@
+
+
+# Cryptocurrency Market App
+
+> A Single Page Application that allows users to track the Cryptocurrency Market.
+> Users can search for country and click on a coin to view their current data of coin stat.
+
+
+## Built With
+
+- JavaScript
+- ReactJS
+- Redux
+- CSS
+
+## Live Demo 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add Future Features and Key Features sections with some features in them to make your readme file more descriptive and professional :blush: